tensorboard/webapp/hparams/_redux/hparams_types.ts

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+/**
+ * @fileoverview Hparams types for usage in reducers and effects.
+ */
+
+import {DiscreteFilter, HparamSpec, IntervalFilter, MetricSpec} from '../types';
+
+/**
+ * Key used to namespace the hparams reducer.
+ */
+export const HPARAMS_FEATURE_KEY = 'hparams';
+
+export type ExperimentId = string;
```

Comment:
I see that this type is a direct copy from runs/, but I don't think it makes sense for us to add a new "public" `ExperimentId`.  Can we drop the "export" on this type?  The runs/ variant of ExperimentId is not used outside of runs_types.ts, afaik.